test/protractor/spec.js

```diff
@@ -8,4 +8,52 @@ describe('PaperHive homepage', function() {
       'PaperHive · The coworking hub for researchers'
     );
   });
+
+    it('should be present subscribe form', function () {
+	var subscribeForm = element(by.name('subscribeForm'));
+	expect(subscribeForm.isPresent()).toBe(true);
+
+    });
+
+    it('should be present about us link', function () {
+	var aboutUs = element(by.linkText('About us'));
+	expect(aboutUs.isPresent()).toBe(true);
+
+	var linkAboutUs = element(by.xpath('/html/body/footer/ng-include/div[1]/div/div/div[2]/ul/li[1]/a'));
+	linkAboutUs.click();
+	expect(browser.getCurrentUrl()).toBe('http://localhost:8080/about');
```

Comment:
I don't  know if I understand the question.
`browser.getCurrentUrl()`

gets the current URL, and then in the expectation the correct URL has to be specified